test/context.js

```diff
@@ -24,8 +24,8 @@ describe('context', function() {
     }
     zmq.Context.setMaxSockets(256);
     zmq.Context.getMaxSockets().should.equal(256);
-    zmq.Context.setMaxSockets(1024);
```

Comment:
Instead of hard-coding the valid sockets num why not save what we have before the change to 256 and then reset.
```javascript
    var currMaxSockets = zmq.Context.getMaxSockets();
    zmq.Context.setMaxSockets(256);
    zmq.Context.getMaxSockets().should.equal(256);
    zmq.Context.setMaxSockets(currMaxSockets);
    done();
```